src/app/backing/BackingPage.tsx

```diff
@@ -44,8 +44,11 @@ export const BackingPage = () => {
   const totalBacking = formatSymbol(totalBackingBigInt, 'stRIF')
   const availableBackingUSD =
     !availableForBackingBigInt || !rifPriceUsd
-      ? formatCurrency(0)
-      : formatCurrency(getFiatAmount(availableForBackingBigInt, rifPriceUsd))
+      ? formatCurrency(0, { currency: 'USD', showCurrency: true })
```

Comment:
I want to discuss the strategy for this, before we had a formatter. Im not sure if I should update it or if we want to create a new one. 

Same scenario that led us to create the formatter. Repetition across multiple components.

Reply:
yes, I think we should use it, as I don't see why we shouldn't.